examples/hooks/FieldHooks/ValidateFieldHooks/formatPhoneNumber.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { Message, TextField, Sheet, Workbook } from '@flatfile/configure'
+import { SheetTester } from '../../../../src/utils/testing/SheetTester'
+
+const testSheet = new Sheet('Test Sheet', {
+  cellPhone: TextField({
+    label: 'Cell Phone Number',
+    validate: (value: string) => {
+      /*the below regex matches these phone number types: 
+      123-456-7890
+      (123) 456-7890
+      123 456 7890
+      123.456.7890
+      +91 (123) 456-7890 */
+      const regex = /^(\+\d{1,2}\s)?\(?\d{3}\)?[\s.-]\d{3}[\s.-]\d{4}$/
+      if (!value.match(regex)) {
+        return [
+          new Message(
+            `${value} is an invalid phone number`,
+            'error',
+            'validate'
+          ),
+        ]
+      }
+    },
+  }),
+})
+
+const TestWorkbook = new Workbook({
+  name: 'Test Workbook',
+  namespace: 'Test',
+  sheets: { testSheet },
+})
+
+describe('Workbook tests ->', () => {
+  // here we use Sheet tester
+  const testSheet = new SheetTester(TestWorkbook, 'testSheet')
+  test('Reformat phone number with international code', async () => {
+    // for this inputRow
+    const inputRow = { cellPhone: '15555555555' }
+    // we expect this output row
+    const expectedOutputRow = { cellPhone: '+1 (555) 555-5555' }
+    const res = await testSheet.testRecord(inputRow)
+    //call the expectation here
+    expect(res).toMatchObject(expectedOutputRow)
+  })
+
+  //additional tests for additional cases
+  test('Remove letters, number too short', async () => {
+    const inputRow = { cellPhone: '12321 ggg' }
+    const expectedOutputRow = { cellPhone: 'Invalid phone number' }
+    const res = await testSheet.testRecord(inputRow)
+    expect(res).toMatchObject(expectedOutputRow)
+  })
+
```

Comment:
@paddymul this hook is waiting on the ability to test the messages that are returned, since it's a Validate hook. That's why my tests are incomplete here.